Raven.Studio.Html5/App/views/manage/cluster.html

```diff
@@ -47,6 +47,8 @@ <h3 class="text-info">
                             <li><a href="#" data-bind="click: initializeNewCluster">Secede from cluster - rude step down into a single node cluster</a></li>
                             <li><a href="#" data-bind="click: addAnotherServerToCluster.bind($root, true)">Kidnap node into cluster - force a node to the current cluster</a></li>
                             <li><a href="#" data-bind="click: removeClustering.bind($root)">Remove cluster entirely and perform cleanup.</a></li>
+                            <li data-bind="visible: !isReplicationCheckEnabled()"><a href="#" data-bind="click: toggleReplicationChecks.bind($root)">Disable replication state sync.</a></li>
+                            <li data-bind="visible: isReplicationCheckEnabled()"><a href="#" data-bind="click: toggleReplicationChecks.bind($root)">Prevent out of date leaders.</a></li>
```

Comment:
Can we replace this with data-bind="text: isReplicationCheckEnabled()? 'Prevent out of date leaders.':'Disable replication state sync' ?